src/pages/AllCategories.jsx

```diff
@@ -16,14 +13,28 @@ const Categories = styled.section`
 `;
 
 export const AllCategories = () => {
+  const [categories, setCategories] = useState([]);
+
+  const getCategories = async () => { 
+    const response = await axios.get('https://africommerce.cyclic.app/categories');
+    setCategories(response.data.categories);
+    console.log(response.data.categories);
+
+  }
+
+  useEffect(() => {
+    getCategories();
+  }, []);
+
+
+
+
   return (
     <Categories>
       <CategoryHeader />
-      <WomenCategory />
-      <MenCategory />
-      <TechCategory />
-      <CarCategory />
-      <KidsCategory />
+      {categories.length > 0 && categories.map((category) => (
```

Comment:
@Sodeeqishola5  can you also add a condition when the category is empty let it show "No category Found"

Reply:
Alright